worlds/apsudoku/docs/setup_en.md

```diff
@@ -13,25 +11,33 @@ Does not need to be added at the start of a seed, as it does not create any slot
 
 ## Installation Procedures
 
-Go to the latest release from the [APSudoku Releases page](https://github.com/EmilyV99/APSudoku/releases). Download and extract the `APSudoku.zip` file.
+Go to the latest release from the [APSudoku Releases page](https://github.com/APSudoku/APSudoku/releases). Download and extract the appropriate file for your platform.
 
 ## Joining a MultiWorld Game
 
-1. Run APSudoku.exe
-2. Under the 'Archipelago' tab at the top-right:
-	- Enter the server url & port number
+1. Run the APSudoku .exe (name varies by platform)
```

Comment:
This should just list them out instead of including a strange space and saying the name varies
```suggestion
1. Run the APSudoku .exe (name varies by platform)
```

Reply:
Then I'd just remove `.exe` here or specify that this is for Windows, the space is confusing and it's not clear that this means the name is _not_ `APSudoku.exe`